doc/manual.rst

```diff
@@ -5513,63 +5489,85 @@ Static params can also appear in the signatures of generic types:
 .. code-block:: nim
 
   type
-    Matrix[M,N: static[int]; T: Number] = array[0..(M*N - 1), T]
+    Matrix[M,N: static int; T: Number] = array[0..(M*N - 1), T]
       # Note how `Number` is just a type constraint here, while
-      # `static[int]` requires us to supply a compile-time int value
+      # `static int` requires us to supply a compile-time int value
 
     AffineTransform2D[T] = Matrix[3, 3, T]
     AffineTransform3D[T] = Matrix[4, 4, T]
 
   var m1: AffineTransform3D[float]  # OK
   var m2: AffineTransform2D[string] # Error, `string` is not a `Number`
 
+Please note that ``static T`` is just a syntactic convenience for the
+underlying generic type ``static[T]``. This means that you can omit the
```

Comment:
> This means

No, it doesn't mean anything like that, it means you can leave out the brackets. That ``static`` is the typeclass of all static types is independent from the syntactic shortcut.

Reply:
Well, my point was to say that it behaves like every other generic type. Just like how `Foo` is equivalent to `Foo[T]`

Reply:
It felt a bit repetitive to make that point for both `static` and `type` types, but I wasn't sure how to communicate that in a different way (perhaps it can sit in the section introducing type classes and implicit generics).